catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/provider/SolrProviderQuery.java

```diff
@@ -1283,6 +1286,52 @@ public void testPropertyIsInProximityTo() throws Exception {
     queryAndVerifyCount(0, getFilterBuilder().proximity(Core.TITLE, 1, "Mary little"), provider);
   }
 
+  @Test()
+  public void testPropertyIsDivibisibleBy() throws Exception {
+
+    deleteAll(provider);
+
+    String longField = "divisibleByCounter";
+    long longFieldValue = 6;
```

Comment:
✏️ Consider moving this next to the metacards below or using literals in `setAttribute()` to make it clear what the values are.